crates/subspace-farmer/src/single_disk_plot/plotting.rs

```diff
@@ -460,7 +464,23 @@ where
             .collect_into(&mut sectors_to_check);
         for (sector_index, history_size) in sectors_to_check.drain(..) {
             if let Some(sector_expire_at) = sectors_expire_at.get(&sector_index) {
-                if *sector_expire_at >= archived_segment_header.segment_index() {
+                trace!(
+                    %sector_index,
+                    %history_size,
+                    %sector_expire_at,
+                    "Checking sector for expiration"
+                );
+                // +1 means we will start replotting a bit before it actually expires to avoid
+                // storing expired sectors
+                if *sector_expire_at
+                    <= (archived_segment_header.segment_index() + SegmentIndex::ONE)
```

Comment:
Can this + 1  be made a method like `inc()`?

Reply:
Possibly, do you think it would be more readable? Rust doesn't have increments and I got used to it eventually.

Reply:
Just preference, this is fine too